assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -61,16 +57,18 @@ export default function useMediaPicker({
   });
   useEffect(() => {
     try {
+      // Handles the video processing logic from WordPress.
+      // The Uploader.success callback is invoked when a user uploads a file.
+      // Race condition concern: please DO NOT upload video poster on this
+      // callback, becuase the video is not guarantee to be ready at the moment.
+      // We should only upload the video poster on the select event.
```

Comment:
Let's make this race condition warning more generic (video posters is too specific).

"Race condition: the video content is not guaranteed to be available in this callback."

Reply:
Got it, updated.